### PR TITLE
#SUP-6410 Related plugin clickUrl modification to open in new tab

### PR DESCRIPTION
**Pull request by ErezBouskila**
@OrenMe

### CODE REVIEW

File: modules/KalturaSupport/components/related/related.js [hunk 1]
```diff
@@ -386,11 +387,15 @@
 
 			if(this.getConfig('clickUrl')){
 				try {
+					if( this.getConfig( 'openInNewTab' ) === true ) {
+						window.open(this.getConfig('clickUrl'), '_blank');
+						return;
```
Comment (OrenMe) [inline modules/KalturaSupport/components/related/related.js]: @ErezBouskila better use if/else and return once.

Comment (omridevk) [inline modules/KalturaSupport/components/related/related.js]: @ErezBouskila , better yet, use Guard Clauses and replace the condition logic, so you can have one return at the top. More info:
http://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Comment (omridevk) [inline modules/KalturaSupport/components/related/related.js]: maybe it won't fit here and you will have to follow Oren's instructions.

Comment (ErezBouskila) [inline modules/KalturaSupport/components/related/related.js]: @OrenMe modified the fix

**Pull request merged**